<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Science Final Project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pNetflix_tab">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Netflix Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Process Raw Netflix Data (This can take a long time)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QGroupBox" name="dataRatingsGroupBox">
             <property name="title">
              <string>Load User Ratings Data (Provided by Netflix)</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="decompressGroupBox">
                <property name="title">
                 <string>Decompress Ratings Data Files</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="decompressButton">
                   <property name="text">
                    <string>Decompress Ratings Data</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="decompressProgressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Parse and Load Decompressed Data</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="LoadButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Load Data</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="loadProgressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="subsetRatingsDataBox">
             <property name="title">
              <string>Instance Selection</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGroupBox" name="reduceMoviesGroup">
                <property name="title">
                 <string>Reduce Movies, removing those with too few ratings</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="reduceMoviesButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Reduce</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="title">
                    <string>Ratings Cutoff</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <widget class="QSpinBox" name="nd_movieRatingsCuttoffSpinBox">
                      <property name="maximum">
                       <number>1000</number>
                      </property>
                      <property name="value">
                       <number>214</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="reduceMoviesProgressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Reduce Users, removing those with too few ratings</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="reduceUsersButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Reduce</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="title">
                    <string>Ratings Cutoff</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <widget class="QSpinBox" name="nd_userRatingsCutoffSpinBox">
                      <property name="maximum">
                       <number>200</number>
                      </property>
                      <property name="value">
                       <number>30</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="reduceUsersProgressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="SRSWRGroupBox">
                <property name="title">
                 <string>Further Reduce Users, using Simple Random Sampling with Replacement (SRSWR)</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QPushButton" name="reduceSRSWRButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Reduce</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="randomSeedGroupBox">
                   <property name="title">
                    <string>Random Seed</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QSpinBox" name="randomSeedSpinBox">
                      <property name="maximum">
                       <number>10000</number>
                      </property>
                      <property name="value">
                       <number>42</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="reduceSRSWRProgressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="saveGroupBox">
             <property name="title">
              <string>Save Reduced Netflix Ratings Data</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QPushButton" name="nd_saveButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="nd_saveProgressBar">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
